### ISSUE TITLE
Extracing full helpers does not work.

### ISSUE THREAD

**Issue by Aranuvir** [makehumancommunity/community-plugins-makeclothes#12]
When setting extract clothes to helpers, the operator does not produce a mesh. (For original bug report see also: http://www.makehumancommunity.org/forum/viewtopic.php?f=3&t=18708#p51708).
BTW, while fixing the bug, we should also make sure the selection which part of the helpers to extract is not empty, or handle an empty selection to avoid python errors. And I'm not sure, if "extract clothes" is the best to describe what the operator actually does...

**Comment by black-punkduck** [comment]
I need to add a line here, because I forgot the #12 in my comment. I did the fix. The "clothes" had been extracted (copied) from the complete mesh, after that everything was deleted, which does not belong to the group(s) selected. I guess in the beginning there was only one group for each vertex. The procedure even worked, when the first group was the one we are looking for. "Helper" was added later I guess. It is in a second group, but since only the first group was tested we got a valid mesh of zero vertices :). I changed the code that all possible groups are checked against the group we want to keep. And "extract clothes" I also changed. These are not always clothes, but these are parts of the helper. So I also wrote, that this might be an optional base for the clothes, which was the intention of this idea imho
